l10n_it_account_tax_kind/tests/test_account_tax_kind.py

```diff
@@ -11,12 +11,6 @@ def setUp(self):
         super(TestAccountTaxKind, self).setUp()
         self.tax_kind_n1 = self.env.ref('l10n_it_account_tax_kind.n1')
 
-    def test_compute_display_name(self):
```

Comment:
Why removing this?

Reply:
@jackjack82 restore the test, moreover you don't need to call `self.tax_kind_n1._compute_display_name()`, just assert that the `display_name` is what we expect it to be